src/lib389/lib389/cli_conf/monitor.py

```diff
@@ -125,16 +126,20 @@ def db_monitor(inst, basedn, log, args):
     report_time = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
     ldbm_mon = ldbm_monitor.get_status()
     dbcachesize = int(ldbm_mon['nsslapd-db-cache-size-bytes'][0])
-    if 'nsslapd-db-page-size' in ldbm_mon:
-        pagesize = int(ldbm_mon['nsslapd-db-page-size'][0])
+    if 'nsslapd-db-mp-pagesize' in ldbm_mon:
```

Comment:
Just for clarity, could you please add a comment with an explanation here in the CLI code that mp = mempool?

Reply:
Done.